message_buffer/programs/message_buffer/src/instructions/resize_buffer.rs

```diff
@@ -82,6 +80,23 @@ pub fn resize_buffer<'info>(
             .realloc(target_size, false)
             .map_err(|_| MessageBufferError::ReallocFailed)?;
     } else {
+        // Check that account doesn't get resized to smaller than the amount of
```

Comment:
Unrelated : why would we want to ever make an account smaller?

Reply:
i don't have a specific use-case other than if we upgraded to something in the future where we didn't need all the space that was currently allocated, then we wouldn't have to pull empty bytes for each `MessageBuffer`